src/compas_timber/connections/t_tenon_mortise.py

```diff
@@ -0,0 +1,246 @@
+from compas_timber._fabrication import Mortise
+from compas_timber._fabrication import Tenon
+from compas_timber._fabrication.btlx_process import TenonShapeType
+from compas_timber.connections.utilities import beam_ref_side_incidence
+
+from .joint import BeamJoinningError
+from .joint import Joint
+from .solver import JointTopology
+
+
+class TTenonMortiseJoint(Joint):
```

Comment:
does this really need to be specific to T?
couldn't we just have a TenonMortiseJoint which you could apply to either T or L? (in terms of this implementation, assuming this is possible regardless of what's in `SUPPORTED_TOPOLOGY`)

Reply:
This is part of a bigger conversation about joints and features. We should definitely get into this in january or february

Reply:
If we go this route, `SUPPORTED_TOPOLOGY` would need to be a `list` to handle multiple connection types. That means I’d also need to update the checks in the **gh components** and the **workflow** to accommodate and process `SUPPORTED_TOPOLOGY` as a `list` instead of a single topology type.

Does everyone agree that I should go ahead with these changes? @chenkasirer  @obucklin

Reply:
if the current implementation of `TTenonMortise` can already handle other topologies (L basically), I'd rename it to `TenonMortise`. I'd also go ahead an make `SUPPORTED_TOPOlOGIES` a list, but it would also be ok for me if we do this in a separate PR as in most cases this is not enforced.  @obucklin what do you think?